### DISCUSSION TITLE
Need update on .Net Maui Stability after .net 7 release

### DISCUSSION THREAD

**Discussion by manishbisen** [dotnet/maui#11331] [General]
After .net 7 release, is maui is enough stable now to develop real world complex apps Instead of hello world or it is still with full of bugs?

**Comment by HausBJB** [comment]
I don't want to offend anyone here, but I don't think it's mature enough yet. Bugs that are very important for the release are postponed. E.g. performance counts. e.g.

e.g. something like https://github.com/dotnet/maui/issues/9485

how can you push something like that in NET8???

I don't know what others think about it, but I can't publish an app that is that slow.

Only C# keeps me at MAUI

**Reply by mrc-eu** [reply]
I never understood the "I know C#" argument. I've also used .NET (C# and VB) for 20+ years, but learning Dart (if you would go to Flutter) is very easy to do if you're an experienced developer.

I honestly think you should choose a product based on if it's mature and/or good, not the language. 

I personally did choose Flutter, just because it's mature, stable and has a lot of support and packages ready to use. Sure, it's not using native controls, but unless what other might say, it produces very fast apps.
